gradle.properties

```diff
@@ -15,3 +15,4 @@ development=true
 kotlin.mpp.stability.nowarn=true
 kotlin.mpp.enableCInteropCommonization=true
 kotlin.mpp.androidSourceSetLayoutVersion=2
+compose.resources.always.generate.accessors=true
```

Comment:
this is not required here. the plugin will generate Res class automatically

Reply:
will drop, thank you